tests/get-coverage.yml

```diff
@@ -0,0 +1,73 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Prepare for coverage extraction
+  hosts: all
+  tasks:
+    - name: set facts
+      set_fact:
+        coverage_module: network_connections
+        coverage: /root/.local/bin/coverage
+    - name: set more facts
+      set_fact:
+        coverage_file: ansible-coverage-{{ coverage_module }}-{{ test_playbook|replace('.yml', '') }}
+
+    - name: set more facts
+      set_fact:
+        destdir: "remote_coverage/{{ test_playbook }}"
+      when: destdir is undefined
+
+    - name: debug info
+      debug:
+        msg: Getting coverage for '{{ coverage_module }}' with '{{ test_playbook }}'
+
+    # combine data in case old data is left there
+    - command: "{{ coverage }} combine"
+      environment:
+        COVERAGE_FILE: "{{ coverage_file }}"
+      ignore_errors: yes
+
+    - name: remove old data
+      file:
+        state: absent
+        path: "{{ item }}"
+      loop:
+        - .coverage
+
+    - name: remove test data
+      shell: rm -rf /root/.ansible/ /root/.coverage.* /root/htmlcov /tmp/ansible_*
+
```

Comment:
can this be merged with `file` above?

Reply:
The problem is the wildcards, AFAICS `with_fileglob` only works locally. I guess we only need `.coverage.*` anymore, the other paths are leftovers from my testing.